### DISCUSSION TITLE
Start manual gallery from X slide

### DISCUSSION THREAD

**Discussion by dandovnar** [fancyapps/ui#191] [Q&A] [answered]
Hello! Is it possible to start gallery, for example, from second slide when it was initialized manually by an array?

Here's my fiddle: https://jsfiddle.net/dandovnar/q428p6mj/

I mean, when I click to the second image the gallery starts from the first image. I would like to synchronize it. Is it possible to do this without Fancybox.getInstance().jumpTo()?

Thanks!

**Comment by fancyapps** [comment, marked as answer]
Hi,

Use `startIndex` option, example:


```js
Fancybox.show(
  [
    { src: "https://lipsum.app/id/1/200x200" },
    { src: "https://lipsum.app/id/2/200x200" },
    { src: "https://lipsum.app/id/3/200x200" },
  ],
  { startIndex: 1 }
);
```

**Reply by dandovnar** [reply]
Thanks!
